frontend/src/pages/org/index.ts

```diff
@@ -73,6 +76,9 @@ export class Org extends LiteElement {
   @property({ type: String })
   orgId!: string;
 
+  @property({ type: Boolean })
+  orgStorageQuotaReached: boolean = false;
```

Comment:
Since this attribute doesn't depend on a reactive property of `btrix-app`, you could make this a private property of `btrix-org`:
```suggestion
  @state()
  private orgStorageQuotaReached = false;
```